### PR TITLE
patch bullet list rendering for clap_mangen

### LINKED ISSUE #5095: test manpage list is broken
see
https://manpages.debian.org/unstable/rust-coreutils/rust-test.1.en.html

to reproduce, with
https://github.com/uutils/coreutils/pull/5092
```

make manpages
man target/debug/man/test.1

```

### PR DESCRIPTION
**Pull request by xuky76**
temporary fix for clap_mangen not rendering bullet lists correctly

image

# Related Issues
- closes #5095

### CODE REVIEW

File: GNUmakefile [hunk 1]
```diff
@@ -340,7 +340,7 @@ distclean: clean
 manpages: build-coreutils
 	mkdir -p $(BUILDDIR)/man/
 	$(foreach prog, $(INSTALLEES), \
-		$(BUILDDIR)/coreutils manpage $(prog) > $(BUILDDIR)/man/$(PROG_PREFIX)$(prog).1; \
+		$(BUILDDIR)/coreutils manpage $(prog) | sed 's/^\* *\(.*\)/.IP \\[bu]\n\1/' > $(BUILDDIR)/man/$(PROG_PREFIX)$(prog).1; \
```
Comment (sylvestre) [inline GNUmakefile:343]: could you please add a comment explaining the why? :)
thanks

Comment (xuky76) [reply GNUmakefile:345]: Update has been made, I had to change the comment to something less descriptive as the spell check lint was unhappy with clap_mangen

Comment (cakebaker) [reply GNUmakefile:345]: @xuky76 an alternative would have been to add the word to the "spell-checker:ignore" line at the top of the file

**Comment by xuky76** [comment]
Not an issue with clap_mangen. Someone should verify but the section that @sylvestre referred to that isn't being parsed correctly is completed by the help_section proc macro and our own parser for markdown. This should probably be fixed up instead of using my quick fix.

```shell
> rg test.md
src/uu/test/src/test.rs
25:const ABOUT: &str = help_about!("test.md");
39:const AFTER_HELP: &str = help_section!("after help", "test.md");
```

**Pull request closed without merging**